components/purchase_modal/index.ts

```diff
@@ -48,6 +49,7 @@ function mapStateToProps(state: GlobalState) {
         invoices: getCloudDelinquentInvoices(state),
         isCloudDelinquencyGreaterThan90Days: isCloudDelinquencyGreaterThan90Days(state),
         isFreeTrial: subscription?.is_free_trial === 'true',
+        isComplianceBlocked: subscription?.compliance_blocked === 'true',
```

Comment:
Is there a related CWS PR that includes this new field?

Reply:
I thought I had requested review on them but I guess I did not. Should be there now.